main/src/ca/uwaterloo/flix/tools/Packager.scala

```diff
@@ -1,23 +1,51 @@
 package ca.uwaterloo.flix.tools
 
-import java.io.PrintWriter
-import java.nio.file.attribute.BasicFileAttributes
-import java.nio.file._
-import java.util.zip.{ZipEntry, ZipFile, ZipOutputStream}
-
 import ca.uwaterloo.flix.api.Flix
 import ca.uwaterloo.flix.language.ast.Ast.Source
 import ca.uwaterloo.flix.runtime.CompilationResult
-import ca.uwaterloo.flix.util.vt.TerminalContext
+import ca.uwaterloo.flix.tools.github.GitHub
 import ca.uwaterloo.flix.util._
+import ca.uwaterloo.flix.util.vt.TerminalContext
 
+import java.io.PrintWriter
+import java.nio.file._
+import java.nio.file.attribute.BasicFileAttributes
+import java.util.zip.{ZipEntry, ZipFile, ZipOutputStream}
 import scala.collection.mutable
 
 /**
   * An interface to manage flix packages.
   */
 object Packager {
 
+  /**
+    * Installs a flix package from the Github `project`.
+    *
+    * `project` must be of the form `<owner>/<repo>`
+    *
+    * The package is installed at `lib/<owner>/<repo>`
+    */
+  def install(project: String, p: Path, o: Options)(implicit tc: TerminalContext): Unit = {
+    val proj = GitHub.parseProject(project)
+    val release = GitHub.getLatestRelease(proj)
+    val assets = release.assets.filter(_.name.endsWith(".fpkg"))
+    val lib = getLibraryDirectory(p)
+    val assetFolder = lib.resolve(proj.owner).resolve(proj.repo)
+
+    // create the asset directory if it doesn't exist
+    Files.createDirectories(assetFolder)
```

Comment:
What folder is this?

Reply:
The asset folder is e.g. `lib/mlutze/cool-thing`. I don't put it directly under `lib` because I think that we shouldn't rely on the file names in the asset. Correct me if I am wrong about that :)

Reply:
OK, I think I understand.

Reply:
This is so you have `lib/mlutze/coolthing/v1.0.1.fpkg`?

Reply:
Yes, or any arbitrary file name (ending in `.fpkg`